examples/subprotocol_server_youi/subprotocol_server_youi.cpp

```diff
@@ -0,0 +1,90 @@
+#include <iostream>
+#include <vector>
+#define ASIO_STANDALONE
+#include <websocketpp/config/asio_no_tls.hpp>
+#include <websocketpp/server.hpp>
+
+typedef websocketpp::server<websocketpp::config::asio> server;
+
+using websocketpp::connection_hdl;
+using websocketpp::lib::placeholders::_1;
+using websocketpp::lib::placeholders::_2;
+using websocketpp::lib::bind;
+using websocketpp::lib::ref;
+
+void print_protocols(std::vector<char*> const &supported_protocols)
```

Comment:
usually we put const at the beginning.
```suggestion
void print_protocols(const std::vector<char*> &supported_protocols)
```